### PR TITLE
Modify the recon_dft_constraint to use child dataset

### PR DESCRIPTION
**Pull request by mathturtle**
For #1301.

### CODE REVIEW

File: tomviz/AddPythonTransformReaction.cxx [hunk 1]
```diff
@@ -358,7 +358,7 @@ OperatorPython* AddPythonTransformReaction::addExpression(DataSource* source)
       addPythonOperator(source, this->scriptLabel, this->scriptSource,
                         arguments);
     }
-  } else if (scriptLabel == "Reconstruct (Constraint-based Direct Fourier)") {
+  }/* else if (scriptLabel == "Reconstruct (Constraint-based Direct Fourier)") {
```
Comment (cryos) [inline tomviz/AddPythonTransformReaction.cxx]: Why not just remove these lines?

Comment (mathturtle) [reply tomviz/AddPythonTransformReaction.cxx]: I think this is one of the reasons this commit was still work in progress before I cherry-picked it...  updated.

**Comment by cryos** [comment]
Any thoughts @ercius or @Hovden on whether this is ready to merge.

**Comment by ercius** [comment]
I just downloaded the nightly (1.1.0-121) but #1301is not fixed. Running the Constraint-based Direct Fourier Method reconstruction does not create a child data set.

I think that all reconstruction methods should behave in the same way (i.e. producing a child data set).

**Comment by cryos** [comment]
I want to document the old UI in my opinion does look better, but I think we need to work in improving possible generated layouts. I think the advantage of uniformity is worth the UI cost for now, but would like to work on improving the generated UI. On my system the original UI:
image
The new UI has a very large minimum size, and a lot of empty space:
image

**Comment by cryos** [comment]
Added some line breaks, and will merge one that is a little improved for layout, we should force the reflow of text in future though.
image

**Pull request merged**